README.md

```diff
@@ -126,7 +126,7 @@ Location | Yes| Primary city in which the system is located, followed by the 2-l
 System ID | Yes | ID for the system. This MUST match the `system_id` field in `system_information.json`.
 URL | Yes | URL for the system from the `url` field in `system_information.json`. If the `url` field is not included in `system_information.json` this SHOULD be the primary URL for the system operator.
 Auto-Discovery URL | Yes | URL for the system's `gbfs.json` auto-discovery file.
-Authentication Info | Conditionally REQUIRED | If authentication is required, this MUST contain a URL to a human-readable page describing how the authentication should be performed and how credentials can be created, or directly contain the public key-value pair to be added to the URL.
+Authentication Info | Conditionally REQUIRED | If authentication is required, this MUST contain a URL to a human-readable page describing how the authentication should be performed and how credentials can be created, or directly contain the public key-value pair to append to the feed URLs.
```

Comment:
This wouldn't be a key-value pair. I think what you want to say here is "contain the API key or token to append to the feed URLs"

Reply:
Thanks. How can we make explicit that the name of the parameter is required too?

Reply:
I think this is getting overly complicated - I don't see any reason to separate the key from the URL in the auto-discovery column. What's the value in that? We've got lots of URLS with parameters for example: `https://qickscooters.rideatom.com/gbfs/v2_2/en/gbfs?id=104` the only difference is that in this case it's an API key not a `system_id` but they function in the same way since the API key doesn't change. It's my understanding that the last column was added to support Auth schemes that use tokens or other forms of Auth that are subject to change and therefore needed a URL to provide additional info on how to get access.

Reply:
The problem here is that `gbfs.json` references the sub-feeds without the key. It needs to be appended manually to them. So I suggest keeping the "key value pair" language.

Reply:
I'm sorry - I misunderstood the problem. Ortwin is correct. I didn't realize that the URLS in `gbfs.json`  lacked the key. However I would also argue that this is broken because  `feeds[].url` in `gbfs.json` should contain a valid URL pointing to the file. Maybe we should consider updating that definition and requiring valid URLs. I've seen this lots of times the past and I just left the feeds out of `systems.csv` until they fixed the `gbfs.json` file.

Reply:
Thank you both for your input!

>I would also argue that this is broken because `feeds[].url` in `gbfs.json` should contain a valid URL pointing to the file.

I agree. I informed the producer that it would be best to remove the public key or append it to all URLs in `gbfs.json`.

>Maybe we should consider updating that definition and requiring valid URLs.

+1